### PR TITLE
[config] Fix naming conflict between 'config syslog' function and 'syslog' module

### LINKED ISSUE #635: config reload/load_minigraph fails with AttributeError: 'Group' object has no attribute 'syslog'
**Description**
config reload/load_minigraph fails with AttributeError: 'Group' object has no attribute 'syslog'

**Steps to reproduce the issue**
1. config load_minigraph -y
2. config reload -y

**Describe the results you received**
```bash
root@sonic:/home/admin# config load_minigraph -y
Traceback (most recent call last):
  File "/usr/bin/config", line 12, in <module>
    sys.exit(config())
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/config/main.py", line 478, in load_minigraph
    log_info("sdfsdfsdf")
  File "/usr/lib/python2.7/dist-packages/config/main.py", line 37, in log_info
    syslog.openlog(SYSLOG_IDENTIFIER)
AttributeError: 'Group' object has no attribute 'openlog'
```

**Describe the results you expected**
```bash
root@sonic:/home/admin# config load_minigraph -y
Stopping service swss ...
Stopping service lldp ...
Stopping service pmon ...
Stopping service bgp ...
Stopping service hostcfgd ...
Running command: /usr/local/bin/sonic-cfggen -H -m -j /etc/sonic/init_cfg.json --write-to-db
Running command: pfcwd start_default
Running command: config qos reload
Running command: /usr/local/bin/sonic-cfggen -d -t /usr/share/sonic/device/x86_64-mlnx_msn3700c-r0/ACS-MSN3700C/buffers.json.j2 >/tmp/buffers.json
Running command: /usr/local/bin/sonic-cfggen -d -t /usr/share/sonic/device/x86_64-mlnx_msn3700c-r0/ACS-MSN3700C/qos.json.j2 -y /etc/sonic/sonic_version.yml >/tmp/qos.json
Running command: /usr/local/bin/sonic-cfggen -j /tmp/buffers.json --write-to-db
Running command: /usr/local/bin/sonic-cfggen -j /tmp/qos.json --write-to-db

Restarting service hostname-config ...
Restarting service interfaces-config ...
Restarting service ntp-config ...
Restarting service rsyslog-config ...
Restarting service swss ...
Restarting service bgp ...
Restarting service pmon ...
Restarting service lldp ...
Restarting service hostcfgd ...
Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`.
```

**Additional information you deem important (e.g. issue happens only occasionally)**
The issue is caused by this PR: https://github.com/Azure/sonic-utilities/pull/609

**Output of `show version`**
```bash
SONiC Software Version: SONiC.HEAD.785-d47ec526
Distribution: Debian 9.10
Kernel: 4.9.0-9-2-amd64
Build commit: d47ec526
Build date: Sun Sep  8 06:49:17 UTC 2019
Built by: johnar@jenkins-worker-7
```

### PR DESCRIPTION
**Pull request by tsvanduyn**
**- What I did**
Updated method name in the config/main.py file to remove the conflict between the "import syslog" command and the click "syslog" command. 

**- How I did it**
Changed the click "syslog" group to be "syslog_group" to remove the conflict

**- How to verify it**
These commands will now work again without the conflict: 
config load_minigraph -y
config reload -y

**- Previous command output (if the output of a command-line utility has changed)**
N/A

**- New command output (if the output of a command-line utility has changed)**
N/A

This fixes #635 
-->

### CODE REVIEW

**Comment by jleveque** [comment]
Retest this please

File: config/main.py [hunk 1]
```diff
@@ -1358,16 +1358,16 @@ def naming_mode_alias():
 #
 # 'syslog' group ('config syslog ...')
 #
-@config.group()
+@config.group('syslog')
 @click.pass_context
-def syslog(ctx):
+def syslog_group(ctx):
```
Comment (nazariig) [inline config/main.py]: @tsvanduyn IMHO, i think we can try to find a better name for this.  

AFAIK, we already have:
```bash
 sudo show logging
```
How about this:
```bash
sudo config logging server add 1.1.1.1
sudo config logging server del 1.1.1.1
sudo show logging servers
```

What do you think?

File: config/main.py [hunk 2]
```diff
@@ -1358,16 +1358,16 @@ def naming_mode_alias():
 #
 # 'syslog' group ('config syslog ...')
```
Comment (nazariig) [inline config/main.py]: @tsvanduyn please align the description

**Comment by nazariig** [comment]
@tsvanduyn please also align the `doc/Command-Reference.md`

File: config/main.py (hunk 1 above)
Comment (jleveque) [reply config/main.py]: The method name has changed, but the command syntax did not, due to the change on line 1361, `@config.group('syslog')`. The subcommand is still named "syslog", and the syntax is still:

```
sudo config syslog server add 1.1.1.1
sudo config syslog server del 1.1.1.1
```

Or do you still prefer `logging` over `syslog`?

File: config/main.py (hunk 2 above)
Comment (jleveque) [reply config/main.py]: The command syntax did not change. Please see my comment below.

**Comment by jleveque** [comment]
@nazariig: The documentation does not need to change. Please see my comments above.

File: config/main.py (hunk 1 above)
Comment (nazariig) [reply config/main.py]: > Or do you still prefer `logging` over `syslog`?
I think we can stay with the original version.

File: config/main.py (hunk 2 above)
Comment (nazariig) [reply config/main.py]: @jleveque thanks for pointing out.

**Pull request merged**